DOCS-5000 Java Module update (#371)
mulesoft/docs-connectors: modules/ROOT/pages/java/java-instanceof.adoc

export resources].
====

To check whether an object belongs to a certain class, an instance of the object is needed.
This can be done by calling the xref:java/java-create-instance.adoc[`new` operation] to create an

> `Employee` object and place it in a variable, in this case, named `jane`:

Reviewer: ```suggestion
`Employee` object and place it in a variable, `jane`:
```

> [TIP]

Reviewer: Review the style reference entry for "Tip notice."

====

> When omitting the `Accept subtypes` parameter it defaults to `true`,

Reviewer: ```suggestion
Omitting the `Accept subtypes` parameter defaults its value to `true`.
```

> the previous code is equivalent to:

Reviewer: Where is the beginning of this sentence?
